### PR TITLE
LUM-18436 - initial commit.

### PR DESCRIPTION
**Pull request by Blackbaud-RajRaulji**
@blackbaud/team-cerebro - Review please. 

Needs some cleanup. Will create another branch with those changes.

### CODE REVIEW

File: reindex-new.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+source "./indexUtil.sh"
+
+# list of fields whose data we want to copy
+documentFieldList=(
+    "id"
+    "tenant_id"
+    "cons_id"
+    "cons_name.first"
```
Comment (Blackbaud-BethRichardson) [inline reindex-new.sh:9]: Will this copy subfields?

File: indexUtil.sh [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+function buildAggregationsQueryUrl() {
+    local _elasticSearchUrl=$1
+    local _env=$2
+    local _idxName=$3
+    local _docType=$4
+
+    if [ "$_env" == 'local' ]; then
+        elasticSearchHostName='http://local.docker:9200'
+    elif [ "$_env" == "oscf-prod" ]; then
+        elasticSearchHostName='http://elasticsearch-6.'"$_env"'.blackbaudcloud.com:9200'
```
Comment (Blackbaud-BethRichardson) [inline indexUtil.sh:125]: Future improvement: pull ports and some of these URL pieces out into one location for easier maintenance.

**Review by Blackbaud-BethRichardson — COMMENTED** [review]
Might be nice to have a readme or some doc notes in each script on usage and intent. Thanks for checking these in!

File: reindex-new.sh (hunk 1 above)
Comment (Blackbaud-KevinEnns) [reply reindex-new.sh:9]: subfields get populated when the main field does, so as long as the subfield is mapped in the new index, it will get populated.

**Review by Blackbaud-KevinEnns — APPROVED** [review]
I'm going to merge these as is.

I think, like @Blackbaud-BethRichardson said, that a bit of a readme or usage/description would help as I had to read quite a bit of bash code before I understood how to use these initially.

**Pull request merged**